queue/src/queue/mod.rs

```diff
@@ -311,7 +317,8 @@ mod tests {
         job_queue
             .with_job(&job1.id, &mut |job| {
                 println!("Making job look old {job:?}");
-                job.updated_at = Utc::now() - chrono::Duration::seconds(ABANDONED_AGE_SECS + 1);
+                job.updated_at =
+                    OffsetDateTime::now_utc() - Duration::seconds(ABANDONED_AGE_SECS + 1);
```

Comment:
[The crate docs](https://docs.rs/time/0.3.17/time/struct.Duration.html) show some magical functions on integers for getting durations, e.g.

```rust
assert_eq!(Duration::ZERO, 0.seconds());
```

However, I am too much of a Rust novice to know how to import that functionality. Pointers welcome.